go/vt/vttablet/tabletmanager/tm_init.go

```diff
@@ -115,6 +118,7 @@ func init() {
 	statsTabletType = stats.NewString("TabletType")
 	statsTabletTypeCount = stats.NewCountersWithSingleLabel("TabletTypeCount", "Number of times the tablet changed to the labeled type", "type")
 	statsBackupIsRunning = stats.NewGaugesWithMultiLabels("BackupIsRunning", "Whether a backup is running", []string{"mode"})
+	statsIsInSRVKeyspace = stats.NewGauge("IsInSRVKeyspace", "Whether the vttablet is in the serving keyspace (1 = true / 0 = false)")
```

Comment:
should is `IsInSrvKeyspace` otherwise the prometheus metric might turn out to be `vttablet_is_in_s_r_v_keyspace`

Reply:
> Should be `IsInSrvKeyspace` otherwise the prometheus metric might turn out to be `vttablet_is_in_s_r_v_keyspace`

Actually I think it is rendered correctly:
```
# HELP vttablet_is_in_srv_keyspace Whether the vttablet is in the serving keyspace (1 = true / 0 = false)
# TYPE vttablet_is_in_srv_keyspace gauge
vttablet_is_in_srv_keyspace 0
```

Reply:
TIL :)